app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -1,5 +1,6 @@
 require 'image-inspector-client'
 require 'kubeclient'
+require 'securerandom'
```

Comment:
@ilackarms I think we should also have an option not to send `auth_token` at all if it is `nil`